internify/src/App.js

```diff
@@ -1,10 +1,30 @@
 import Landing from "./old-components/pages/Landing";
 import Login from "./old-components/pages/Login";
-import "./App.css";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
 
 function App() {
   return (
-    <Login/>
+    <Router>
+       <Switch>
+          <Route path="/home">
+            <Login />
+          </Route>
+          <Route path="/login">
+            <Login />
+          </Route>
+          <Route path="/create">
+           <h2>Hello from create page</h2>
+          </Route>
+          <Route path="/">
+            <Login />
+          </Route>
+        </Switch>
+    </Router>
   );
```

Comment:
NOTE: these are placeholders until the other pages have been merged into develop